htmd/qm/custom.py

```diff
@@ -97,6 +97,8 @@ def minimize(self, coords, restrained_dihedrals):
 
             for dihedral in restrained_dihedrals:
                 theta0 = dihedralAngle(coords[dihedral])
+                # TODO: This is a work-around. jit signatures should be fixed instead so theta0 is never a ndarray
```

Comment:
@stefdoerr can you look at this work-around and see if the jit signatures can solve it? I don't know if you remember it.

Reply:
works fine on loro so I'm removing the work-around on my own branch to see if I catch it